tests/nightly/security-internal-apparmor/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Check internal vendored apparmor in snapd snap
+
+manual: true
+
+environment:
+    AAP: /var/lib/snapd/apparmor/profiles/snap.test-snapd-setpriority.test-snapd-setpriority
+
+prepare: |
+    echo "Install a helper snap with default confinement"
+    snap install test-snapd-setpriority
+
+    echo "Enable installing the snapd snap, this happens automatically"
+    snap set core experimental.snapd-snap=true
+    # give the state time to create the change and then start watching
+    # it
+    retry -n 30 snap watch --last=transition-to-snapd-snap
```

Comment:
this is also unnecessary, you can just do `snap install snapd --edge` these days and it "just works" everywhere